book/src/design/scoping.md

```diff
@@ -56,14 +53,11 @@ Trade-offs
 Goals which we currently believe are not safely achievable in this first deployment without losing some of the above goals:
 * Improving Zcash's bandwidth (number of transactions per time) or latency (time for a transaction).
 * Deploying cross-chain interoperation such as the Inter-Blockchain Communication protocol (IBC) or Cross-Chain Interoperability Protocol (CCIP).
-* Supporting a large number of finalizer so that any user who wants to run their own finalizer can do so.
-* Supporting casual users, with minimal computer expertise, running finalizers.
+* Supporting a large number of finalizers so that any user who wants to run their own finalizer can do so.
+* Supporting casual users, with little computer expertise, running finalizers.
 
 Non-Goals
 ---
 
-* The consensus mechanism is needed only to prevent double-spending/multi-spending/rollback attacks. It is not needed for censorship-resistance, since that is provided by end-to-end encryption, and it is not needed for counterfeiting-resistance, since that is provided by two layers of defense: proofs and turnstiles. So censorship-resistance and counterfeiting-resistance are non-goals for (at least) the first deployment of Crosslink.
-* This is neutral with regard to Zcash governance -- it doesn't change anything about Zcash governance.
-* This does not change the emissions schedule -- how much total ZEC is in circulation at any given point in time in the future -- and in particular it does not change the eventual 21 million ZEC supply cap.
-
-See also [Zebra Crosslink Implementation Requirements](https://docs.google.com/document/d/1YXalTGoezGH8GS1dknO8aK6eBFRq_Pq8LvDeho1KVZ8/edit?usp=sharing).
+* The consensus mechanism is needed primarily to prevent double-spending/multi-spending/rollback attacks. It is not the primary provider of censorship-resistance, since that is provided primarily by end-to-end encryption, and it is not the primary provider of counterfeiting-resistance, since that is provided by proofs and turnstiles. So censorship-resistance and counterfeiting-resistance are not primary goals for (at least) the first deployment of Crosslink. Hm, except actually we _do_ need censorship-resistance for transactions that have transparent components or that a would-be censor would have out-of-band knowledge of. So revisit this and consider converting it into a (limited) goal instead of a non-goal.
```

Comment:
```suggestion
* The consensus mechanism is needed primarily to prevent double-spending/multi-spending/rollback attacks. It is not the primary provider of censorship-resistance, since that is provided primarily by end-to-end encryption, and it is not the primary provider of counterfeiting-resistance, since that is provided by proofs and turnstiles. So censorship-resistance and counterfeiting-resistance are not primary goals for (at least) the first deployment of Crosslink.
```